docs/designers-developers/faq.md

```diff
@@ -267,39 +267,39 @@ We plan to continue supporting existing meta boxes while providing new ways to e
 
 The main extension point we want to emphasize is creating new blocks. We are still working on how to extend the rest of the UI that is built in JS. We are tracking it here: [Issue #1352](https://github.com/WordPress/gutenberg/issues/1352)
 
-## Will Custom Post Types be supported?
+## Are Custom Post Types still supported?
 
 Indeed. There are multiple ways in which custom post types can leverage Gutenberg. The plan is to allow them to specify the blocks they support, as well as defining a default block for the post type. It's not currently the case, but if a post type disables the content field, the “advanced” section at the bottom would fill the page.
 
-## Will there be columns?
+## Does Gutenberg support columns?
 
 Yes, a columns block is available in Gutenberg.
 
-## Will there be nested blocks?
+## Does Gutenberg support nested blocks?
 
 We are currently implementing the infrastructure for nested blocks support. We expect this to open further customization opportunities. Block authors also can nest components and HTML inside of a block during construction. The UI for manipulating nested blocks is still being refined, and depending on the timing, it might not be included in the first version of Gutenberg.
```

Comment:
This answer will probably need a rewrite; nested blocks are live and in Gutenberg.

Reply:
What about:

```suggestion
Yes, it is supported. You can have multiple levels of nesting – blocks within blocks within blocks.
```

Reply:
Looks good. Can you include it in this PR?